mealie/routes/recipe/recipe_crud_routes.py

```diff
@@ -459,7 +459,8 @@ def update_last_made(self, slug: str, data: RecipeLastMade):
         """Update a recipe's last made timestamp"""
 
         try:
-            recipe = self.service.update_last_made(slug, data.timestamp)
+            # If timestamp is None, update with None
+            recipe = self.service.update_last_made(slug, data.timestamp if data.timestamp is not None else None)
```

Comment:
Wouldn't 
```suggestion
            recipe = self.service.update_last_made(slug, data.timestamp)
```
make more sense? If it's !!None, doesn't that mean it's None?